src/pages/TeamSettings/CloudHooks.vue

```diff
@@ -354,6 +354,7 @@ export default {
           prevIcon: 'keyboard_arrow_left',
           nextIcon: 'keyboard_arrow_right'
         }"
+        :loading="!cloudHooks"
```

Comment:
I'm not sure the lack of `cloudHooks` means the component is "loading". I would use the more typical approach of defining a `loadingKey` on the apollo query (line 227) and defining loading as `loadingKey > 0`

Reply:
@stackoverfloweth I added the `loadingKey` option to the query but it's not updating the data property that I passed it. When I reload the page, the property stays at 0 but I'm not sure why? I found that we already had a `isLoadingTable` computed property and used that instead which sets the return value to `true` in the result hook. Is that okay too?

Reply:
Hmm yeah, in my testing I had the same result. That's really weird that this query doesn't seem to work well with the standard `loadingKey` like the others. The thing I don't like about using the `isLoadingTable` is that it too just becomes false only when there are results, so if there were no items I believe it would appear to load forever.  But after spending a good hour or so with no better solution I'm fine with just moving forward as is